services/sentry/sentry/sentry.conf.py

```
@@ -0,0 +1,72 @@
+        },
+    }
+}
+
+# Prefer SENTRY_SYSTEM_SECRET_KEY; fall back to SENTRY_SECRET_KEY for compatibility
+SENTRY_SYSTEM_SECRET_KEY = os.environ.get(
+    "SENTRY_SYSTEM_SECRET_KEY",
+    os.environ.get("SENTRY_SECRET_KEY", ""),
+)
+SECRET_KEY = SENTRY_SYSTEM_SECRET_KEY
+
+SENTRY_OPTIONS["system.secret-key"] = SENTRY_SYSTEM_SECRET_KEY
```

Ruff (0.15.9)

[error] `SENTRY_OPTIONS` may be undefined, or defined from star imports

(F405)